packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/ssr.js

```diff
@@ -9,7 +9,6 @@
 
 const path = require('path')
 const {getRuntime} = require('@salesforce/pwa-kit-runtime/ssr/server/express')
-const {isRemote} = require('@salesforce/pwa-kit-runtime/utils/ssr-server')
```

Comment:
Why does the retail react app use import, but the generated overrides use require?

Reply:
Good question. I'd like to know this as well